jerry-debugger/jerry-client-ws.py

```diff
@@ -932,6 +940,22 @@ def main():
 
             prompt.cmdloop()
 
+        elif buffer_type == JERRY_DEBUGGER_MEMSTATS_RECEIVE:
+
+            memory_stats = struct.unpack(debugger.byte_order + debugger.idx_format
+                                        + debugger.idx_format + debugger.idx_format
+                                        + debugger.idx_format + debugger.idx_format
+                                        + debugger.idx_format,
+                                        data[3: 3 + 4 + 4 + 4 + 4 + 4 + 4])
+
+            print ("Allocated bytes: " + str(memory_stats[0]))
```

Comment:
I think you used use the % format:

`"Allocated bytes: %d" % (str(memory_stats[0]))`